rclcpp/src/rclcpp/node.cpp

```diff
@@ -242,6 +242,31 @@ Node::get_parameters(
   return results;
 }
 
+const rclcpp::parameter::ParameterVariant
+Node::get_parameter(const std::string & name) const
+{
+  rclcpp::parameter::ParameterVariant parameter;
+
+  if (get_parameter(name, parameter)) {
+    return parameter;
+  } else {
+    throw std::out_of_range("Parameter: " + name + " does not exist in map");
```

Comment:
nitpick: would you change this to `"Parameter '" + name + "' not set"`? We usually wrap variables output into string inside of quotes in case `name` is an empty string, for example. Also, the fact that a map is used is an implementation detail.